Comp-409/notes.md

```diff
@@ -774,15 +775,20 @@ class Thread {
 
 enter:
     me.locked = true        // signifies to others that they should be locked
+    myPred = TS(tail,me)    // set the tail to my node
     while (myPred.locked)   // spin
 
 exit:
     me.locked = false
     me = myPred
 ```
-
-* Notice that nodes get moved around, and there's no spin on exit
+Property
+* Notice that nodes get moved around
+* No spin on exit
 * Still FCFS
+* Cache Friendly
+* Space Efficient
+* Only require test and set (TS)
```

Comment:
require`s`; I would also write `Only requires TS` to stay consistent, and because people can look back at the glossary.